aion/web/elm/src/View.elm

```diff
@@ -21,62 +20,66 @@ view model =
         [ page model ]
 
 
-layout : Html Msg -> Location -> Navbar.State -> Html Msg
-layout content location navbarState =
-    div
-        [ class "layout" ]
-        [ navbar location navbarState
-        , content
+layout : Html Msg -> Route -> Location -> Navbar.State -> Html Msg
+layout content route location navbarState =
+    div []
+        [ navbar route location navbarState
+        , div
+            [ class "layout" ]
+            [ content ]
+        , customFooter footerContent
         ]
 
 
-navbar : Location -> Navbar.State -> Html Msg
-navbar location navbarState =
-    Navbar.config NavbarMsg
-        |> Navbar.withAnimation
-        |> Navbar.success
-        |> Navbar.container
-        |> Navbar.brand
-            [ href "#" ]
-            [ img
-                [ src ((host location) ++ "svg/hemp.svg")
-                , class "header-aion-logo"
-                ]
-                []
-            , text "Aion"
-            ]
-        |> Navbar.items
-            [ Navbar.itemLink [ href roomsPath ] [ text "Rooms" ]
-            , Navbar.itemLink [ href userPath ] [ text "Profile" ]
-            ]
-        |> Navbar.customItems
-            [ Navbar.customItem
-                (Button.button
-                    [ Button.roleLink
-                    , Button.onClick Logout
+customFooter : List String -> Html Msg
+customFooter footerContent =
+    footer []
+        (List.map
+            (\paragraph -> p [] [ text paragraph ])
+            footerContent
+        )
+
+
+navbar : Route -> Location -> Navbar.State -> Html Msg
+navbar route location navbarState =
+    let
+        baseNavbar =
+            Navbar.config NavbarMsg
+                |> Navbar.withAnimation
+                |> Navbar.success
+                |> Navbar.container
+                |> Navbar.brand
+                    [ href "#" ]
+                    [ img
+                        [ src ((host location) ++ "images/aion_logo.png")
```

Comment:
Building urls as separate let statements seems to be the good practice. So you could move the `(host location ++ ...)` part as the first let statement and here just put the variable name.